templates/microRTPS_agent.cpp.em

```diff
@@ -114,6 +114,7 @@ struct options {
 static void usage(const char *name)
 {
     printf("usage: %s [options]\n\n"
+             "  -a <whitelist>          List of IP numbers of whitelisted interfaces\n"
```

Comment:
What is the format of the list? IP addresses separated with spaces or commas?

Reply:
The help text shows what was intended, but the implementation currently just supports one ip address :) I suggest having comma-separated list and finalizing the implementation in a way that it actually takes in the list.